src/components/Buildbaron/BBFIleTicket.tsx

```diff
@@ -0,0 +1,151 @@
+      setCreatedTicketsCount={setCreatedTicketsCount}
+      createdTicketsCount={createdTicketsCount}
+    />
+  </>
+);
+
+interface FileTicketProps {
+  taskId: string;
+  dispatchBanner;
+  setCreatedTicketsCount;
+  createdTicketsCount: number;
+}
+
+export const FileTicket: React.FC<FileTicketProps> = ({
+  taskId,
+  dispatchBanner,
+  setCreatedTicketsCount,
+  createdTicketsCount,
+}) => {
+  const [fileJiraTicket, { loading: loadingFileJiraTicket }] = useMutation<
+    BbCreateTicketMutation,
+    BbCreateTicketMutationVariables
+  >(FILE_JIRA_TICKET, {
+    onCompleted: () => {
+      setButtonText("FILE ANOTHER TICKET");
+      dispatchBanner.successBanner(
+        `Ticket successfully created for this task.`
+      );
+      setCreatedTicketsCount(createdTicketsCount + 1);
+    },
+    onError(error) {
+      dispatchBanner.errorBanner(
+        `There was an error filing the ticket: ${error.message}`
+      );
+    },
+  });
+
+  const [buttonText, setButtonText] = useState<string>("FILE TICKET");
+  const onClickFile = () => {
+    // todo: add analytics
+    fileJiraTicket({ variables: { taskId } });
+  };
+
+  return (
+    <Popconfirm
+      title="Do you want to create a failure ticket for this task?"
+      onConfirm={onClickFile}
+      icon={null}
+      placement="right"
+      okText="File Ticket"
+      okButtonProps={{ loading: loadingFileJiraTicket }}
+      cancelButtonProps={{ disabled: loadingFileJiraTicket }}
+    >
+      <FileButton
+        dataCy="file-ticket-button"
+        variant={Variant.Primary}
+        size="small"
+      >
+        {buttonText}
+      </FileButton>
+    </Popconfirm>
+  );
+};
+
+interface CreatedTicketsProps {
+  taskId: string;
+  dispatchBanner;
+  setCreatedTicketsCount;
+  createdTicketsCount;
+}
+
+export const CreatedTickets: React.FC<CreatedTicketsProps> = ({
+  taskId,
+  dispatchBanner,
+  setCreatedTicketsCount,
+  createdTicketsCount,
+}) => {
+  const { data, startPolling, stopPolling } = useQuery<
+    GetCreatedTicketsQuery,
+    GetCreatedTicketsQueryVariables
+  >(GET_CREATED_TICKETS, {
+    variables: { taskId },
+    onError(error) {
+      dispatchBanner.errorBanner(
+        `There was an error getting tickets created for this task: ${error.message}`
+      );
+    },
+  });
+  const length = data?.bbGetCreatedTickets?.length;
+
+  if (createdTicketsCount > data?.bbGetCreatedTickets?.length) {
+    startPolling(1 * 1000);
+  } else {
+    setCreatedTicketsCount(length);
+    stopPolling();
+  }
+
+  return (
+    <>
+      {length > 0 && (
+        <>
+          <TitleAndButtons>
+            <BBTitle>Tickets Created From This Task </BBTitle>
+          </TitleAndButtons>
+          <BuildBaronTable jiraIssues={data?.bbGetCreatedTickets} />{" "}
+        </>
+      )}
+    </>
+  );
+};
+
+const FileButton = styled(Button)`
+  justify-content: center;
+  // width: 97px;
+  height: 22px;
+  margin-bottom: 15px;
+  font-size: 11px;
```

Comment:
Maybe try using a different variant of the Leafygreen Button instead of setting the font size and changing its style to much

Reply:
I don't see a variant that would fit what I need. Did you have one in mind?

Reply:
![image](https://user-images.githubusercontent.com/13104717/93813874-c0fabe80-fc21-11ea-870f-630c93ca5167.png)
this is what it looks like without the styling I added. 

this is what I need it to look like: 
![image](https://user-images.githubusercontent.com/13104717/93813949-dec82380-fc21-11ea-9432-f8d76b3d0005.png)


I don't think I can achieve it by using a different variant of the Leafygreen Button.

Reply:
I would try changing the `size` prop for the button to make the button smaller instead of changing the font-size through css additions
https://github.com/mongodb/leafygreen-ui/tree/master/packages/button#properties